packages/ts-exporter/pom.xml

```diff
@@ -0,0 +1,128 @@
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.appformer</groupId>
+  <artifactId>ts-exporter</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>AppFormer JS :: API TypeScript Exporter</name>
+  <description>TypeScript API exporter for AppFormer.js</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.testing.compile</groupId>
+      <artifactId>compile-testing</artifactId>
+      <version>0.15</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.3.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.5</version>
+    </dependency>
+    <dependency>
+      <artifactId>guava</artifactId>
+      <groupId>com.google.guava</groupId>
+      <version>15.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
+      <version>0.9.11</version>
+    </dependency>
+  </dependencies>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.0</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+          <proc>none</proc>
+        </configuration>
+      </plugin>
+    </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.4.5</version>
+          <configuration>
+            <reportsDirectory>target/pit-reports</reportsDirectory>
+            <timestampedReports>true</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <mutators>
+              <!-- See http://pitest.org/quickstart/mutators/ -->
+              <mutator>DEFAULTS</mutator>
+              <mutator>NON_VOID_METHOD_CALLS</mutator>
+              <mutator>REMOVE_CONDITIONALS</mutator>
+            </mutators>
+            <avoidCallsTo>java.lang.StringBuilder</avoidCallsTo>
```

Comment:
I've just realized there is a small issue with this `<avoidCallsTo>`. It overrides the default avoided calls which exclude logging calls by default. My intention was not to override that, but to exclude calls to `StringBuilder` in addition to logging calls. What I intended was:
```xml
<avoidCallsTo>
  <avoidCallsTo>java.lang.StringBuilder</avoidCallsTo>
  <avoidCallsTo>org.slf4j</avoidCallsTo>
</avoidCallsTo>
```
Please change it to the above (option A) or simply remove the line (option B).

## Option A - avoid logging calls and String concatenation mutations
Java String concatenation is compiled to `StringBuilder.append()` calls (at least that's what the Pitest report shows). In my project, String concatenation is almost exclusively used for exception message construction and `Object.toString()` implementation. I've made the decision not to test exception messages in general and `Object.toString()`.

## Option B - only avoid logging calls mutations
You may want to give it a try and see for yourself if the analysis of string concatenation mutations has a value for you or if it just generates noise.

Mutations of logging calls will be avoided in both options and it's most probably what you want. Testing logging calls rarely makes sense (I've never seen/needed to do it).